providers/summary/scancode.js

```diff
@@ -84,10 +91,26 @@ class ScanCodeSummarizer {
     if (!licenses) return null
     const list = setToArray(licenses)
     if (!list) return null
-    return list
-      .map(normalizeSpdx)
-      .filter(x => x)
-      .join(' AND ')
+    if (list.length === 1) return list[0]
+    return list.map(x => `(${x})`).join(' AND ')
+  }
+
+  _createExpression(rule, licenseKey) {
```

Comment:
name is misleading. this function is selecting which value to use (expresion or key) and cleaning up (normalizing) the expression if that's that direction we picked.

Reply:
👍 agreed - don't want to leave like this